src/components/ui/CheckboxGroup/index.tsx

```diff
@@ -0,0 +1,88 @@
+import { useId, type FC, useCallback } from 'react';
+
+import { Checkbox } from '@/components/ui/Checkbox';
+import styles from '@/components/ui/CheckboxGroup/index.module.css';
+
+import type { ChangeEvent } from 'react';
+
+type Option = {
+  label: string;
+  value: string;
+  disabled?: boolean;
+};
+
+type Props = {
+  label: string;
+  name: string;
+  description?: string;
+  value: string[];
+  options: Option[];
+  required?: boolean;
+  errorMessage?: string;
+  direction?: 'column' | 'row';
+  onChange: (value: string[]) => void;
+};
+
+export const CheckboxGroup: FC<Props> = ({
+  label,
+  name,
+  description,
+  value,
+  options,
+  required,
+  errorMessage,
+  direction = 'column',
+  onChange,
+}) => {
+  const id = useId();
+  const isError = !!errorMessage;
+
+  const handleChange = useCallback(
+    (e: ChangeEvent<HTMLInputElement>) => {
+      console.log(e.target.value);
+      let newValue: string[];
+      if (e.target.checked) {
+        newValue = value.concat([e.target.value]);
+      } else {
+        newValue = value.filter((value) => value !== e.target.value);
+      }
+      onChange(newValue);
+    },
+    [value, onChange],
+  );
```

Comment:
immutable っぽい感じでも良さそう

```suggestion
  const handleChange = (e: ChangeEvent<HTMLInputElement>) => {
    const newValue = e.target.checked
      ? [...value, e.target.value]
      : value.filter((val) => val !== e.target.value);
    onChange(newValue);
  };
```

Reply:
@jp-knj 

あ、間違い無いです😇
ここリファクタしてpushしまする